docs/build/tools/foundry.mdx

```diff
@@ -0,0 +1,246 @@
+---
+description: A guide for Foundry
+---
+
+# Foundry
+
+[Foundry] is a smart contract development environment
+for EVM-based chains. This guide will show you how to use Foundry to build,
+test, and deploy smart contracts on Oasis Sapphire.
+
+:::info
+
+For comprehensive details about Foundry's features, consult the
+[Foundry documentation].
+
+:::
+
+Foundry contains the following tools:
+
+- **Forge** is a CLI [tool] for building, testing,
+and deploying smart contracts.
+Due to integrated revm it significantly increases runtime speed when testing.
+
+- **Anvil** is a local development EVM [node].
+It is installed as part of Foundry, but currently cannot
+be extended with Sapphire features.
+
+- **Cast** is a [CLI tool] for interacting with EVM nodes.
+It uses RPC calls, so it can be used
+to interact with Sapphire nodes running the [Oasis Web3 gateway].
+
+- **Chisel** is a Solidity [REPL] (short for "read-eval-print loop")
+that allows developers to write and test Solidity code snippets.
+It provides an interactive environment for writing and executing
+Solidity code, as well as a set of built-in commands for working
+with and debugging your code.
+
+
+## Setup and Configuration
+
+Foundry uses default revm EVM implementation which does not contain
```

Comment:
Let's move this paragraph under the `Installing Sapphire Foundry (Optional)` section.

Also:

```suggestion
Foundry uses [revm] EVM implementation which does not contain
```

and below:
```
[revm]: https://github.com/bluealloy/revm
```

Reply:
Updated docs.